com.obsproject.Studio.Plugin.NDI.json

```diff
@@ -0,0 +1,132 @@
+{
+    "app-id": "com.obsproject.Studio.Plugin.NDI",
+    "runtime": "com.obsproject.Studio//stable",
```

Comment:
```suggestion
    "runtime": "com.obsproject.Studio",
    "runtime-version": "stable",
```

Reply:
this is I think the build error. (it's cryptic)

Reply:
The `error.txt` makes it less cryptic from what I read from it.